edx_event_bus_kafka/consumer/settings/common.py

```diff
@@ -0,0 +1,17 @@
+"""Default settings values."""
+
+
+def plugin_settings(settings):
```

Comment:
Instead of this, I think we'll just want an `edx_event_bus_kafka.config` module that reads producer and consumer settings into a dict or dataclass or whatever, and the producer and consumer code both rely on that. (It can default to devstack settings or just have empty defaults.)